### PR TITLE
test(menulist, message): minor changes related to BDD steps FE-2352

### PR DESCRIPTION
**Pull request by KatarzynaQA**
### Proposed behaviour

Amend Given-When-Then steps for menuList and Message component. Amend scenario related to outline color for alert, alertClassic, dialog, dialogClassic, dialogFullScreen, menu components

### Checklist

<del> - [ ] Release notes (Conventional Commits)  <del>
<del>  - [ ] Unit tests <del>  
- [x] Cypress automation tests

<del>  - [ ] Storybook added or updated <del>  
<del>  - [ ] Theme support <del>  
<del>  - [ ] Typescript `d.ts` file added or updated <del>  

### Additional context

No need to add other tests for menuList and message components due to lack of difference between default and classic stories.

### Testing instructions

run: npm run test-cypress and choose proper feature file

### CODE REVIEW

File: cypress/features/regression/menuList.feature [hunk 1]
```diff
@@ -1,49 +1,49 @@
...
+      | title                   |
+      | Sample text             |
+      | 1234567890              |
+      | áéíóú¿¡üñ               |
+      | !@#$%^*()_+-=~[];:.,?{} |
+      | ÄÖÜßäöüß                |
+      | <>                      |
 
   @ignore
   # @positive doesn't work on Carbon Demo site
```
Comment (DlgSHi) [inline cypress/features/regression/menuList.feature]: is it still not working?

File: cypress/features/regression/menuList.feature [hunk 2]
```diff
@@ -54,27 +54,27 @@ Feature: Menulist component
...
   Scenario: Enable and disable innitially open
-    When I check initiallyOpen checkbox
-      And I uncheck initiallyOpen checkbox
+    Given I check initiallyOpen checkbox
```
Comment (DlgSHi) [inline cypress/features/regression/menuList.feature]: did we check the other ignored tags ?

File: cypress/support/step-definitions/common-steps.js [hunk 3]
```diff
@@ -393,12 +393,8 @@ Then('closeIcon is not visible', () => {
...
 });
 
-Then('closeIcon has the border outline', () => {
-  closeIconButton().should('have.css', 'outline', 'rgb(255, 181, 0) solid 3px');
-});
-
-Then('closeIcon has border outline for classic story', () => {
-  closeIconButton().should('have.css', 'outline', 'rgba(0, 103, 244, 0.247) auto 5px');
+Then('closeIcon has the border outline color {string}', (color) => {
+  closeIconButton().should('have.css', 'outline-color', color);
```
Comment (DlgSHi) [inline cypress/support/step-definitions/common-steps.js]: shouldn't we check also border width?

File: cypress/features/regression/menuList.feature (hunk 1 above)
Comment (KatarzynaQA) [reply cypress/features/regression/menuList.feature]: yes, it doesn't work

Comment (KatarzynaQA) [reply cypress/features/regression/menuList.feature]: yes, doesn't work

File: cypress/features/regression/menuList.feature (hunk 2 above)
Comment (KatarzynaQA) [reply cypress/features/regression/menuList.feature]: it doesn't work

File: cypress/support/step-definitions/common-steps.js (hunk 3 above)
Comment (KatarzynaQA) [reply cypress/support/step-definitions/common-steps.js]: fixed

**Pull request merged**